Makefile

```diff
@@ -5,16 +5,18 @@
 ##                             VERSION PARAMS                                 ##
 ################################################################################
 
-## Docker Build Versions
-DOCKER_BUILD_IMAGE = golang:1.17
-DOCKER_BASE_IMAGE = alpine:3.16
-
 ## Tool Versions
+GOLANG_VERSION := $(shell cat go.mod | grep "^go " | cut -d " " -f 2)
```

Comment:
@gabrieljackson  Would like feedback on this. I'm like 1/5, since I'd rather be explicit but on the other hand I can't find why we would use a different version on the Makefile than in the Go file.

Reply:
Hey, yeah I think I am 0/5 on this. If we didn't have to still explicitly update the circleci config as well then I could see this making more sense. I am totally fine with either implementation so I will leave it up to you.